unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/ProfanityFiltering/Resources/Profanity/badwords.json

```diff
@@ -28,7 +28,6 @@
     "4r5e",
     "5h1t",
     "5hit",
-    "a55",
```

Comment:
Cannot we just have it as an explicit word?

Reply:
Sure, that would work aswell